### PR TITLE
Pootle fs boilerplate

### PR DESCRIPTION
**Pull request by phlax**
Adds boilerplate code required for pootle_fs

- apps.py
- signals
- delegates
- exceptions

### CODE REVIEW

File: pootle/apps/pootle_fs/exceptions.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) Pootle contributors.
+#
+# This file is a part of the Pootle project. It is distributed under the GPL3
+# or later license. See the LICENSE file for a copy of the license and the
+# AUTHORS file for copyright and authorship information.
```
Comment (unho) [inline pootle/apps/pootle_fs/exceptions.py]: What is the point of this empty file?

**Comment by unho** [comment]
Other than the nit, lgtm.

File: pootle/apps/pootle_fs/exceptions.py (hunk 1 above)
Comment (phlax) [inline pootle/apps/pootle_fs/exceptions.py]: good point, ill remove

**Comment by julen** [comment]
As mentioned the other day, I'm pretty much with a strong -1 on merging anything fs related until at least some important master regressions are fixed (and hence can be deployed).

**Comment by phlax** [comment]
@julen this is a core feature for the 2.8 release, and we are landing this asap.

Re. the regressions, there are a couple but they are pretty minor, and relatively trivial to fix.

If they are not fixed by the time i have finished landing this stuff (and the related documentation) i will address them.

**Pull request merged**